### PR TITLE
machinegroup: wrap dnode callbacks

### PR DESCRIPTION
**Pull request by rjeczalik**
Fixes:

```
$ kd exec @5834df10a4deb2d329fecfe2 echo abc
error executing "exec" command: interface conversion: dnode.caller is dnode.functionReceived, not dnode.callback (f7bdab253fe8446e)
```

### CODE REVIEW

**Comment by rjeczalik** [comment]
@usirin have you seen this?

```
server.handlers.invitetoteambycsv should be able to process 500 items at once:
     Uncaught AssertionError: expected [Error: invitation is not found] to not exist
```

**Comment by usirin** [comment]
Nope, but where do you think I would have seen this?

**Comment by cihangir** [comment]
@usirin webserver test failed on that step

File: go/src/koding/klient/machine/machinegroup/exec.go [hunk 1]
```diff
@@ -40,7 +42,31 @@ func (r *ExecRequest) Valid() error {
 	if err := r.ExecRequest.Valid(); err != nil {
 		return err
 	}
-	return r.MachineRequest.Valid()
+	if err := r.MachineRequest.Valid(); err != nil {
+		return err
+	}
+
+	// dnode.Function cannot be forwarded, they need to be
```
Comment (cihangir) [inline go/src/koding/klient/machine/machinegroup/exec.go]: 🤔

**Review by cihangir — APPROVED** [review]

**Review by ppknap — APPROVED** [review]

**Pull request merged**